tests/playwright/assets/api-requests.ts

```diff
@@ -1,177 +1,139 @@
-				force: 1,
+	public async create( request: APIRequestContext, entity: string, data: Post ) {
+		const response = await request.post( `${ this.baseUrl }/index.php`, {
+			params: { rest_route: `/wp/v2/${ entity }` },
+			headers: {
+				'X-WP-Nonce': this.nonce,
 			},
-		} ) );
+			multipart: data,
+		} );
+
+		if ( ! response.ok() ) {
+			throw new Error( `
+				Failed to create a ${ entity }: ${ response.status() }.
+				${ await response.text() }
+				${ response.url() }
+				TEST_PARALLEL_INDEX: ${ process.env.TEST_PARALLEL_INDEX }
+				NONCE: ${ this.nonce }
+			` );
+		}
+		const { id } = await response.json();
+
+		return id;
 	}
-	await Promise.all( requests );
-}
 
-export async function cleanUpTestPages( request: APIRequestContext ) {
-	const pagesPublished = await getPages( request ),
-		pagesDraft = await getPages( request, 'draft' ),
-		pages = [ ...pagesPublished, ...pagesDraft ];
+	public async createMedia( request: APIRequestContext, image: Image ) {
+		const imagePath = image.filePath;
+		const response = await request.post( `${ this.baseUrl }/index.php`, {
 
-	const pageIds = pages
-		.filter( ( page: WpPage ) => page.title.rendered.includes( 'Playwright Test Page' ) )
-		.map( ( page: WpPage ) => page.id );
+			params: { rest_route: '/wp/v2/media' },
+			headers: {
+				'X-WP-Nonce': this.nonce,
+			},
+			multipart: {
+				file: fs.createReadStream( imagePath ),
+				title: image.title,
+				status: 'publish',
+				description: image.description,
+				alt_text: image.alt_text,
+				caption: image.caption,
+			},
+		} );
 
-	for ( const id of pageIds ) {
-		await deletePage( request, id );
-	}
-}
+		if ( ! response.ok() ) {
+			throw new Error( `
+			Failed to create default media: ${ response.status() }.
+			${ await response.text() }
+		` );
+		}
 
-export async function createApiContext( request: APIRequest,
-	options: { storageStateObject: string| StorageState, wpRESTNonce: string, baseURL: string } ) {
-	const context = await request.newContext( {
-		baseURL: options.baseURL,
-		storageState: options.storageStateObject,
-		extraHTTPHeaders: {
-			'X-WP-Nonce': options.wpRESTNonce,
-		},
-	} );
-
-	return context;
-}
+		const { id } = await response.json();
 
-async function _delete( request: APIRequestContext, entity: string, id: string ) {
-	const response = await request.delete( '/index.php', {
-		params: { rest_route: `/wp/v2/${ entity }/${ id }` },
-		headers,
-	} );
+		return id;
+	}
 
-	if ( ! response.ok() ) {
-		throw new Error( `
-			Failed to delete a ${ entity }: ${ response.status() }.
-			${ await response.text() }
-		` );
+	public async deleteMedia( request: APIRequestContext, ids: string[] ) {
+		const requests = [];
+		for ( const id in ids ) {
+			requests.push( request.delete( `${ this.baseUrl }/index.php`, {
+				headers: {
+					'X-WP-Nonce': this.nonce,
+				},
+				params: {
+					rest_route: `/wp/v2/media/${ ids[ id ] }`,
+					force: 1,
+				},
+			} ) );
+		}
```

Comment:
Please wrap this block with empty lines.

Reply:
Fixed: https://github.com/elementor/elementor/pull/27669/commits/c46c0be3ca9d25f0b0eb82f5e26563fad49206ea